snapshot/lib/snapshot/options.rb

```diff
@@ -137,6 +137,13 @@ def self.available_options
                                      description: "Should the project be cleaned before building it?",
                                      is_string: false,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :test_without_building,
+                                     short_option: "-T",
+                                     env_name: "SNAPSHOT_TEST_WITHOUT_BUILDING",
+                                     description: "Test without building, requires a derived data path",
```

Comment:
Can we add a check for this in `snapshot/detect_values.rb` and show a nice `UI.user_error!("...")` if that's not the case, with instructions on how to set it up?